extensions/wikia/MercuryApi/MercuryApiController.class.php

```diff
@@ -433,6 +433,13 @@ public function getPage() {
 				if ( !$isMainPage ) {
 					$titleBuilder->setParts( [ $data['article']['displayTitle'] ] );
 				}
+			} else if ( !$article instanceof Article ) {
```

Comment:
We use `elseif` above. Let's stick to one of them (I prefer `else if`, but whatever).